compiler/input.stanza

```diff
@@ -517,12 +524,14 @@ defn to-iimport (e:IImportExp) -> IImport :
          + te
 
       imp("import form") :
-         IImportExp: {package:v prefix:(pr ...)}
+         IImportExp: {package:v directives:(pr ...)}
 
       pr("prefix form") :
```

Comment:
The "prefix form" string is used for error messages. So we can change that to "directives" now.

Reply:
Add either a check or a consolidation operation for combining 'only' lists.